website/common/script/content/mystery-sets.js

```diff
@@ -304,6 +25,17 @@ const mysterySets = {
     end: '2014-04-01',
   },
 };
+const FIRST_MYSTERY_SET = moment('2014-02-01');
+
+for (let mysteryMonth = FIRST_MYSTERY_SET;
```

Comment:
I got slightly confused by the indentation here :P I wasn't exactly sure what was going on, could we change it to something like:

```
for (
    let mysteryMonth = FIRST_MYSTERY_SET;
    ...
) {
...
}
```

?

Reply:
Heh, lint's indentation rules are not flawless for ensuring readability 😸 Will fix!